aks-node-controller/parser/helper.go

```diff
@@ -48,6 +48,12 @@ var (
 	containerdConfigTemplate = template.Must(
 		template.New("containerdconfigforaksnodeconfig").Funcs(getFuncMapForContainerdConfigTemplate()).Parse(containerdConfigTemplateText),
 	)
+	//go:embed  templates/containerd_no_GPU.toml.gtpl
+	containerdConfigNoGPUTemplateText string
+	//nolint:gochecknoglobals
+	containerdConfigNoGPUTemplate = template.Must(
+		template.New("containerdconfigforaksnodeconfig").Funcs(getFuncMapForContainerdConfigTemplate()).Parse(containerdConfigNoGPUTemplateText),
```

Comment:
It's probably doesn't matter, but `containerdconfigforaksnodeconfig` template name is already used few lines above.

Maybe it's better to pass empty string instead?

Reply:
Don't know if the names matter. Updated with 2 different and descriptive names.